akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcSpec.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.jdbc
+
+import akka.actor.testkit.typed.scaladsl.LogCapturing
+import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import com.typesafe.config.Config
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class JdbcSpec(config: Config) extends ScalaTestWithActorTestKit(config) with AnyWordSpecLike with LogCapturing
```

Comment:
I think I had a comment suggesting that we should remove JdbcSpec, so that it's easier for users to look at and copy from our tests

Reply:
I had it to another PR (#210) and I'm keeping there. Will come.